wasm/wasm_source/Cargo.lock

```diff
@@ -165,9 +168,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.55"
+version = "1.0.58"
```

Comment:
We should revert these unrelated dependency updates back to whatever version is in where we're trying to integrate with in `master` (so `v0.6.1`). But I think it's fine for this PR since we want to get CI passing ASAP, will open another PR once this PR lands and CI is fixed for `eth-bridge-integration` to tidy up.